adagrams/game.py

```diff
@@ -1,11 +1,149 @@
+import random
+import copy
+from webbrowser import get
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+    'A': 1, 
+    'B': 3, 
+    'C': 3, 
+    'D': 2, 
+    'E': 1, 
+    'F': 4, 
+    'G': 2, 
+    'H': 4, 
+    'I': 1, 
+    'J': 8, 
+    'K': 5, 
+    'L': 1, 
+    'M': 3, 
+    'N': 1, 
+    'O': 1, 
+    'P': 3, 
+    'Q': 10, 
+    'R': 1, 
+    'S': 1, 
+    'T': 1, 
+    'U': 1, 
+    'V': 4, 
+    'W': 4, 
+    'X': 8, 
+    'Y': 4, 
+    'Z': 10
+}
+
 def draw_letters():
-    pass
+    '''This function returns 10 random letter tiles. Frequency of letters determined by LETTER_POOL'''
+    letters = []
+    letter_pool_copy = copy.deepcopy(LETTER_POOL)
+    while len(letters) < 10:
+        random_letter = random.choice(list(letter_pool_copy))
+        if letter_pool_copy[random_letter] == 0:
+            continue
+        else:
+            letters.append(random_letter)
+            letter_pool_copy[random_letter] -= 1
+    return letters
+
+    '''PSUEDOCODE:
+    1) Make deep copy of LETTER_POOL
+    2) if LETTER_POOL.deepcopy value = 0; remove
+        3) Generate random letter from current LETTER_POOL.deepcopy- 
+        4) Find letter in LETTER_POOL.deepcopy 
+        5) -1 from value
+        6) append letter to letters[]
+    7) Return letters[]'''
```

Comment:
I love that y'll started with pseudocode to think through the logic! That said, it is also good practice to clean up our comments to have only necessary information for future readers of the code! 😊